x/storage/client/cli/tx.go

```diff
@@ -670,20 +672,34 @@ func CmdUpdateGroupMember() *cobra.Command {
 
 func CmdPutPolicy() *cobra.Command {
```

Comment:
We can refactor it by passing a policy json to make a better user experience.

It's not urgent. we can do it in another pr.